content/en/service_management/events/pipelines_and_processors/category_processor.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Category Processor
+kind: Documentation
+
+---
+
+Use the category processor to add a new attribute (without spaces or special characters in the new attribute name) to a event matching a provided search query. Then, use categories to create groups for an analytical view (for example, URL groups, machine groups, environments, and response time buckets).
+
+**Notes**:
+
+* The syntax of the query is the one in the Event Explorer search bar. This query can be done on any event attribute or tag, whether it is a facet or not. Wildcards can also be used inside your query.
+* Once the event has matched one of the processor queries, it stops. Make sure they are properly ordered in case a event could match several queries.
```

Comment:
```suggestion
* Once the event has matched one of the processor queries, it stops. Make sure they are properly ordered in case an event could match several queries.
```